apis/vshn/v1/dbaas_vshn_postgresql.go

```diff
@@ -124,4 +124,12 @@ type VSHNPostgreSQLBackup struct {
 type VSHNPostgreSQLStatus struct {
 	// PostgreSQLConditions contains the status conditions of the backing object.
 	PostgreSQLConditions []v1.Condition `json:"postgresqlConditions,omitempty"`
+	NamespaceDebug       []v1.Condition `json:"namespaceDebug,omitempty"`
+	ProfileDebug         []v1.Condition `json:"profileDebug,omitempty"`
+	PGConfigDebug        []v1.Condition `json:"pgconfigDebug,omitempty"`
+	PGClusterDebug       []v1.Condition `json:"pgclusterDebug,omitempty"`
+	SecretsDebug         []v1.Condition `json:"secretDebug,omitempty"`
+	S3BucketDebug        []v1.Condition `json:"s3BucketDebug,omitempty"`
+	S3BackupConfigDebug  []v1.Condition `json:"s3BackupConfigDebug,omitempty"`
+	NetworkPolicyDebug   []v1.Condition `json:"networkPolicyDebug,omitempty"`
```

Comment:
I think it's a bit weird to call them `Debug` why not `Conditions` like the `PostgreSQLConditions`?

Reply:
because it's main purpose is to help our customers debug their instances, this is why I wanted to include keyword `debug` in json, then I just named variable consistently

Reply:
Well the `PostgreSQLCondition` is basically the same thing and is named differently. Also basically no field in k8s is called `debug`, but :shrug: